### ISSUE TITLE
rapid-photo-downloader does not run

### ISSUE THREAD

**Issue by dcarosone** [NixOS/nixpkgs#210262]
### Describe the bug
the rapid-photo-downloader python tool does not start, and emits the following error:

```
❯ rapid-photo-downloader
Traceback (most recent call last):
  File "/nix/store/gpyb7lgvkiks5sh1bg0g9cqnz700vv7f-rapid-photo-downloader-0.9.18/bin/.rapid-photo-downloader-wrapped", line 6, in <module>
    from raphodo.rapid import main
  File "/nix/store/gpyb7lgvkiks5sh1bg0g9cqnz700vv7f-rapid-photo-downloader-0.9.18/lib/python3.10/site-packages/raphodo/rapid.py", line 62, in <module>
    gi.require_version('Notify', '0.7')
  File "/nix/store/1psjx5nf2gglgd59cjrnqkqpxyy5qy9p-python3.10-pygobject-3.42.2/lib/python3.10/site-packages/gi/__init__.py", line 126, in require_version
    raise ValueError('Namespace %s not available' % namespace)
ValueError: Namespace Notify not available
```

### Steps To Reproduce
1. run current `nixos-unstable` with package installed
2. invoke rapid-photo-downloader

### Additional context

There are comments in the package suggesting this is a known, solved issue:

``` nix
  # NOTE: Without gobject-introspection in buildInputs, launching fails with
  #       "Namespace [Notify / GExiv2 / GUdev] not available"

```

The package hasn't changed recently, and worked a while ago, so this regression has come about via changes elsewhere. I have seen this before, and it 'fixed itself' on a later rebuild. I didn't look into what changed at the time, but that too was after the most recent update of this file in nixpkgs. That suggests to me that the "fix" via gobject introspection is either incomplete or unstable, based on additional dependencies, … ?

The package also disables tests:
``` nix
  # Package has no generally usable unit tests.
  # The included doctests expect specific, hardcoded hardware to be present.
  doCheck = false;
```

which means that this can't be picked up when it happens due to some other change.  It might be useful to add a minimal test to see whether it can run and produce usage output or similar.

Aside, the package is also out of date; I tried a trivial update-version-and-hash to see if that would help anything, but it then failed for an additional dependency on `show-in-file-manager` that's been added in the meantime, and is not in nixpkgs. There are also probably others, several are mentioned in the changelog. I don't know anything about python so stopped there for now to not get the two issues conflated, but nevertheless an update might move or remove this problem too. There are lots of fixes listed in the changelog since the current nixpkgs version which is from Jan 2020, including one that mentions gobject introspection.

### Notify maintainers
@jfrankenau  @ryantm 

### Metadata

 - system: `"x86_64-linux"`
 - host os: `Linux 5.15.86, NixOS, 23.05 (Stoat), 23.05pre441271.c07552f6f7d`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.12.0`
 - channels(root): `"nixos"`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos`

```

**Comment by wegank** [comment]
Should be closed by #207166.

**Comment by wegank** [comment]
Oh, this is a duplicate of #208249. Closing now.
